elements/radio-group/radio-group.js

```diff
@@ -1,151 +1,233 @@
+      let hasCheckedChild = this.querySelector('[role="radio"][tabindex="0"]');
+      if (!hasCheckedChild) {
+        this.querySelector('[role="radio"]').setAttribute('tabindex', 0); 
+      }
+      this.addEventListener('keydown', this._onKeyDown);
+      this.addEventListener('click', this._onClick);
+    }
 
-  }
+    /**
+     * If the `RadioGroup` is removed from the DOM, clean up any event
+     * listeners.
+     */
+    disconnectedCallback() {
+      this.removeEventListener('keydown', this._onKeyDown);
+      this.removeEventListener('click', this._onClick);
+    }
 
-  onKeyDown(e) {
-    switch (e.keyCode) {
-      case KEYCODE.UP:
-      case KEYCODE.LEFT:
-        {
-          // We will be doing all the handling, so we don’t want
-          // the browser to do anything for us.
+    /**
+     * If the user pressed an arrow key, call preventDefault to prevent the
+     * page from scrolling. If the up or left arrow keys were pressed, select
+     * the previous `RadioButton`. If the down or right keys were pressed,
+     * select the next `RadioButton`.
+     */
+    _onKeyDown(e) {
+      switch (e.keyCode) {
+        case KEYCODE.UP:
+        case KEYCODE.LEFT:
           e.preventDefault();
-          _this.setCheckedToPrevButton();
+          this._setCheckedToPrevButton();
           break;
 
-        }
-
-      case KEYCODE.DOWN:
-      case KEYCODE.RIGHT:
-        {
-          // Same as above.
+        case KEYCODE.DOWN:
+        case KEYCODE.RIGHT:
           e.preventDefault();
           this._setCheckedToNextButton();
           break;
-        }
-
+        
+        default:
+          break;
+      }
     }
-  }
-
-  get selectedRadioButton() {
-    return this.querySelector('[aria-checked="true"]') || this.firstRadioButton;
-  }
 
-  get lastRadioButton() {
-    return Array.from(this.querySelectorAll('[role="radio"]')).pop();
-  }
+    /**
+     * A getter for whichever `RadioButton` is currently checked. If no button
+     * is checked, return the first `RadioButton` child.
+     */
+    get checkedRadioButton() {
+      return this.querySelector('[aria-checked="true"]') || this.firstRadioButton;
+    }
 
-  get firstRadioButton() {
-    return Array.from(this.querySelectorAll('[role="radio"]')).shift();
-  }
+    /**
+     * A getter for the first `RadioButton` child.
+     */
+    get firstRadioButton() {
+      return Array.from(this.querySelectorAll('[role="radio"]')).shift();
+    }
 
-  // Return the radio button that is next to the given one.
-  _prevRadioButton(node) {
-    let prev = node.previousElementSibling;
+    /**
+     * A getter for the last `RadioButton` child.
+     */
+    get lastRadioButton() {
+      return Array.from(this.querySelectorAll('[role="radio"]')).pop();
+    }
 
-    while (prev) {
-      if (prev.getAttribute('role') === 'radio') {
-        return prev;
+    /**
+     * Return the previous sibling of the node that was passed in. The
```

Comment:
The comments on the following functions are kinda missing a first line that explains what the function does on a higher level. For example for this one I’d expect something like “Returns the radio button coming before the one passed as an argument”.